object-filter/src/main/java/org/infinispan/objectfilter/impl/aggregation/AvgAccumulator.java

```diff
@@ -0,0 +1,51 @@
+package org.infinispan.objectfilter.impl.aggregation;
+
+/**
+ * @author anistor@redhat.com
+ * @since 8.0
+ */
+public final class AvgAccumulator extends FieldAccumulator {
+
+   private static class Avg {
```

Comment:
No, but I see what you mean. Handling those pesky not-numbers... I'll address this in another PR.